src/main/java/uk/gov/hmcts/reform/managecase/client/datastore/CaseUserRole.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.managecase.client.datastore;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+@AllArgsConstructor
+public class CaseUserRole {
+
+    @JsonProperty("case_id")
+    private String caseId;
+    @JsonProperty("user_id")
+    private String userId;
+    @JsonProperty("case_role")
+    private String caseRole;
```

Comment:
obviously a very minor matter, but could we pls put and empty line between all subsequent class members?